src/tests/AccountCreation.test.js

```diff
@@ -0,0 +1,35 @@
+import '@testing-library/jest-dom/extend-expect'
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { postCheckingAccounts } from '../api';
+import {CheckingAccountCreation} from '../components/CheckingAccountCreation'
+
+//this test is not finished yet
+jest.mock('../api/index.js');
+
+test('Should create the checking account with the given info',async()=>{
+
+    const response = {
+      status:200
+    }
+
+    postCheckingAccounts.mockResolvedValue(response); 
+    
+    const form = render(<CheckingAccountCreation/>)
+
+    const name = form.getByTestId('account-name');
+    const deposit = form.getByTestId('initial-deposit');
+    const submit = form.getByTestId('submit'); 
+
+    fireEvent.change(name,{target:{value:'testing'}}); 
+    fireEvent.change(deposit,{target:{value:'2500'}}); 
```

Comment:
There may be cases where you need to work on more complex/long tests and use data more than once. I suggest you create an object containing the data, so that you can reuse it and easily change it.

Reply:
Yes, I have to improve that. Because as you said I probably will use it more than once